app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -22,17 +22,15 @@ import com.google.android.material.navigation.NavigationView
 
 
 class MainActivity : AppCompatActivity() {
-
-
     private lateinit var appBarConfiguration: AppBarConfiguration
     private lateinit var binding: ActivityMainBinding
 
-    val allActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
+    private val allActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
```

Comment:
This is not used anywhere else, so I changed this to private

Reply:
Not used anywhere at all?  If that's the case, I'd remove it entirely.

You can verify whether or not something is used by selecting "Find Usages" in Android Studio.